index.html

```diff
@@ -37,7 +38,180 @@ <h1 class="header"><span>Hey there.
+                        <li><img src="img/rails.png" alt="ruby"></li>
+                        <li><img src="img/rspec.png" alt="Rspec"></li>
+                        <li><img src="img/capy.png" alt="Capybara"></li>
+                        <li><img src="img/selenium.png" alt="Selenium"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item-1">
+                <div>
+                    <div id="three"></div>
+                    <h4 class="header-3">Skills</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/database.png" alt="database"></li>
+                        <li><img src="img/git.png" alt="git"></li>
+                        <li><img src="img/cli.png" alt="cli"></li>
+                        <li><img src="img/web.png" alt="web"></li>
+                        <li><img src="img/api.png" alt="api"></li>
+                    </ul>
+                </div>
+            </div>
+            <div>
+                <h1 class="header-4">I'm always interested in hearing about new projects, so if you'd like to chat
+                    please get in touch.</h1>
+            </div>
+            <div>
+                <form class="form">
+                    <div class="form-group">
+                        <input class="input-text" type="text" id="first-name" name="first-name" placeholder="Full name"
+                            required>
+                    </div>
+                    <div class="form-group">
+                        <input class="input-text" type="email" id="email" name="email" placeholder="Email address"
+                            required>
+                    </div>
+                    <div class="form-group">
+                        <textarea class="input-message" id="connect" name="connect"
+                            placeholder="Hey I'd like to connect to..."></textarea>
+                        <img class="union" src="img/Union.png">
+                    </div>
+                    <div class="form-group">
+                        <button class="btn-2" type="button"><span class="text">Get in touch</span></button>
+                    </div>
+                </form>
+            </div>
+            <div>
+                <h1 class="email">ngachaian@gmail.com</h1>
+            </div>
+            <div>
+                <ul class="images-2">
+                    <li><a href=""><img src="img/twitter.png" alt="Twitter"></a></li>
+                    <li> <a href=""><img src="img/Linkedin icon.png" alt="Linkedin"></a></li>
+                    <li><a href=""> <img src="img/medium.png" alt="Medium"></a></li>
+                    <li><a href=""><img src="img/github.png" alt="Github"></a></li>
+                    <li><a href=""><img src="img/angellist.png" alt="Angellist"></a></li>
+                </ul>
+            </div>
+            <div id="four"></div>
+        </section>
```

Comment:
You did great in going a bit further with your project, however i'd advise that you always try to stick to the milestone given for submission. In your case you have part of the next milestone which is the `contact me` section in this milestone, so i'd kindly suggest that you create a new branch and copy this part of your project to it for submission ofr the next milestone 👍